components/exercises/chapter-4-oscar/AList/prepend.tsx

```diff
@@ -0,0 +1,15 @@
+import { List } from './CityList';
+
+/**
+ * Prepends a value to a list.
+ * @param value - The value to prepend to the list.
+ * @param list - The list to which the value will be prepended.
+ * @returns A new list with the value prepended to it.
+ */
+export function prepend(
+  value: number | string,
+  list: List | null
+): List | null {
```

Comment:
I think the return type is wrong, this function can never return `null`.

Reply:
I have removed the type and so far works good, thank you